Algorithm.Framework/Portfolio/SymbolData.cs

```diff
@@ -0,0 +1,71 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using QuantConnect.Data;
+using QuantConnect.Data.Consolidators;
+using QuantConnect.Indicators;
+
+namespace QuantConnect.Algorithm.Framework.Portfolio
+{
+    /// <summary>
+    /// Contains data specific to a symbol required by this model
+    /// </summary>
+    public class SymbolData
```

Comment:
`SymbolData` is a "generic" name we use in many models, so a more specific name would be welcome. For example, `RateOfChangeSymbolData` or `ReturnSymbolData`.